common/src/objects/stored_workload_spec.rs

```diff
@@ -38,6 +38,8 @@ pub struct StoredWorkloadSpec {
     pub runtime_config: String,
     #[serde(default)]
     pub control_interface_access: ControlInterfaceAccess,
+    #[serde(default)]
```

Comment:
Do we need ordered output here as well?
```suggestion
    #[serde(default, serialize_with = "serialize_to_ordered_map")]
```